benchmarks/benchmark_filters.py

```diff
@@ -0,0 +1,16 @@
+# See "Writing benchmarks" in the asv docs for more information.
```

Comment:
Which asv docs is this referring to?

Reply:
@stefanv good point! They are here:

http://asv.readthedocs.io/en/latest/writing_benchmarks.html

@KimNewell would you mind adding this line to your PR:

```
# http://asv.readthedocs.io/en/latest/writing_benchmarks.html
```

You edit the file to add that line and then `git commit -a` and `git push`, and this PR will be automatically updated. 😊